splash/qtrender_image.py

```diff
@@ -67,10 +68,17 @@ def qimage_to_pil_image(self, qimage):
         buf = qimage.bits().asstring(qimage.byteCount())
         if sys.byteorder != "little":
             buf = self.swap_byte_order_i32(buf)
-        return Image.frombytes(
-            self.pillow_image_format,
-            self._qsize_to_tuple(qimage.size()),
-            buf, 'raw', self.pillow_decoder_format)
+        # PIL>2.0 doesn't have fromstring. But older ones don't have frombytes.
+        try:
+            return Image.frombytes(
+                self.pillow_image_format,
+                self._qsize_to_tuple(qimage.size()),
+                buf, 'raw', self.pillow_decoder_format)
+        except AttributeError:
+            return Image.fromstring(
+                self.pillow_image_format,
+                self._qsize_to_tuple(qimage.size()),
+                buf, 'raw', self.pillow_decoder_format)
```

Comment:
I think it is better to do something like https://github.com/kmike/psd-tools/blob/7efb5c6342373bf5a0f663892de98f0fb07f33db/src/psd_tools/user_api/pil_support.py#L14 to avoid duplicating all the arguments